components/LandingPage.js

```diff
@@ -5,19 +5,19 @@ import normalize from 'react-native-normalize';
 
 const LandingPage = ({ navigation }) => {
   return (
-    <View style={styles.container}>
+    <View style={styles.container} testID="lp-view">
       <Header
         containerStyle={{ backgroundColor: '#483D8B', justifyContent: 'space-around', paddingBottom: 10}}
         centerComponent={{ text: 'Never Alone', style: { color: '#fff', fontSize: normalize(30), fontWeight: "bold" } }}
       />
       <View style={styles.box}>
         <Text style={styles.text}>Choose Your Role:</Text>
-        <TouchableOpacity style={styles.careTakerButton} onPress={() => navigation.navigate('Caretaker')} >
+        <TouchableOpacity style={styles.careTakerButton} testID="lp-caretaker-button" onPress={() => navigation.navigate('Caretaker')} >
```

Comment:
This testID (and the one below) are used for unit testing. I think they were two of only a few (maybe only two) unit tests.